### PR TITLE
Add Commishes scraper

### PR DESCRIPTION
**Pull request by faucetlol**
Adds a scraper for the Commishes portfolio.

Did this quite a while ago, but I was having problems with redirects, which now seems to be resolved with 4fab87691ac35d1f0e68b113bba9421f958ae509.

### CODE REVIEW

**Comment by Earlopain** [comment]
Why close this?

**Comment by faucetlol** [comment]
Not intentionally, probably a misclick.

File: app/logical/sites/definitions/commishes.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+type: ScraperDefinition
 enum_value: commishes
 display_name: Commishes
 homepage: https://commishes.com
 gallery_templates:
   - portfolio.commishes.com/user/{site_artist_identifier}
   - ych.commishes.com/user/{site_artist_identifier}
+  - ping.commishes.com/@{site_artist_identifier}
 username_identifier_regex: "[a-zA-Z0-9_-]{3,20}"
+submission_template: https://portfolio.commishes.com/upload/show/{site_submission_identifier}/
```
Comment (Earlopain) [inline app/logical/sites/definitions/commishes.yml:10]: Missing newline, then I'll merge

Comment (faucetlol) [reply app/logical/sites/definitions/commishes.yml]: Weird, VSCode is supposed to do that for me. It also inexplicably changed the line endings to CRLF (despite running on Arch Linux) but Rubocop at least caught that one before I pushed it. Either way, done.

**Comment by Earlopain** [comment]
Thanks! Perhaps I'll look into a yml linter

File: app/logical/sites/definitions/commishes.yml (hunk 1 above)
Comment (Earlopain) [reply app/logical/sites/definitions/commishes.yml]: Should be caught by CI in the future https://github.com/Earlopain/reverser/pull/97

**Pull request merged**